cmd/replication/main.go

```diff
@@ -97,7 +97,8 @@ func main() {
 			}()
 		}
 
-		if options.Replication.Enable || options.Sync.Enable || options.Indexer.Enable {
+		if options.Replication.Enable || options.Sync.Enable || options.Indexer.Enable ||
```

Comment:
this means that the migrator needs to have the Signer.PrivateKey of the node. Otherwise we will compute a different namespace. Do we want to pass this info around?

Reply:
It's needed, as the migrator has to sign envelopes PayerEnvelope and OriginatorEnvelope both.

Reply:
do we expect to have a separate payer key for this? Or are we reusing the node key?

Reply:
Separate payer key, already set up in testnet. That way we can properly identify the migrator "payer" if needed.

Reply:
the signer.private key should be checked in `validateMigrationOptions`

Reply:
Yes, that's done. Maybe I explained myself wrong.
The migrator has two keys:
- Separate payer key, to properly identify the migrator as an unique payer.
- Node private key, which currently is the same as node 100. It's the same key where the payer runs, and the infra reflects this.

Reply:
makes sense. Can you add these keys to `validateMigrationOptions`? Approving otherwise